users/dlip/gamepad.h

```diff
@@ -0,0 +1,181 @@
+// Copyright 2023 Dane Lipscombe (@dlip)
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Code forked from https://github.com/brentaro/QMK_HID_Gamepad_with_SOCD
+#pragma once
+#include "keycodes.h"
+#include "quantum_keycodes.h"
+#include "joystick.h"
+
+enum gamepad_keycodes { GP_DPU = SAFE_RANGE, GP_DPD, GP_DPL, GP_DPR, GP_X, GP_A, GP_B, GP_Y, GP_LB, GP_LT, GP_LSB, GP_RB, GP_RT, GP_RSB, GP_STA, GP_BCK, GP_HOM };
```

Comment:
You now have 2 blocks of code using `SAFE_RANGE`, which will not work.

Reply:
Is there any better approaches than just adding a number to it?

Reply:
I'll put a `custom_keycodes.h` file in my other pr